doc/elements/pagination.md

```diff
@@ -0,0 +1,19 @@
+# Pagination
+
+We have also default pagination:
+<div class="pagination">
+  <a href="#" class="separator"><</a>
+  <a href="#" class="current">1</a>
+  <span>...</span>
+  <a href="#" class="total">10</a>
+  <a href="#" class="separator">></a>
+</div>
+```html
+<div class="pagination">
+  <a href="#" class="separator"><</a>
+  <a href="#" class="current">1</a>
+  <span>...</span>
+  <a href="#" class="total">10</a>
+  <a href="#" class="separator">></a>
+</div>
```

Comment:
Question is - how to properly write this in docs?
Or we suggest that users of this system definitely use Aurelia?